.drone.yml

```diff
@@ -37,13 +37,26 @@ steps:
     commands:
       - make lint
 
-  - name: build
+  - name: e2e
     image: golang:1.13
     commands:
-      - make build
+      - make e2e-test
     volumes:
       - name: deps
         path: /go
+    environment:
+      MCLI_PUBLIC_API_KEY:
+        from_secret: mcli_public_api_key
```

Comment:
[q] Did this end up targeting cloud dev/QA in the end?

Reply:
had a chat with @jamesbroadhead today we are now doing against dev